lib/cocoapods/installer.rb

```diff
@@ -453,6 +453,17 @@ def ensure_plugins_are_installed!
       end
     end
 
+    DEFAULT_PLUGINS = { 'cocoapods-stats' => {} }
```

Comment:
Could we get an explicit test to check that the stats plugin is enabled by default? And also to ensure its gone when the disable stats end variable is set (unless it's handled elsewhere, but I feel this is the place).